lib/sidekiq/group/collection.rb

```diff
@@ -27,9 +27,9 @@ def add(jid)
         Sidekiq::Logging.logger.info "Scheduling child job #{jid} for parent #{@cid}" if Sidekiq::Group.debug
 
         Sidekiq.redis do |r|
-          r.multi do
-            r.sadd("#{@cid}-jids", jid)
-            r.expire("#{@cid}-jids", CID_EXPIRE_TTL)
+          r.multi do |pipeline|
```

Comment:
https://github.com/mperham/sidekiq/issues/5202

`Warning: Redis Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0`